index.html

```diff
@@ -46,6 +51,14 @@ <h2> About </h2>
         <a href="#ref:1">[1]</a> and <a href="#ref:2">[2]</a> for examples of
         on-screen layouts. </p>
 
+        <p> The recommendations in this document represent the consensus that has
+        emerged among Cree-speaking communities over many years of typing Cree
+        syllabics on keyboards with various platforms. They are the result of a
+        long-term collaboration between Cree-speaking communities and the
+        <a href="https://altlab.artsrn.ualberta.ca/">Alberta Language Technology Lab</a>,
+        which focuses on research and development of language technology for
+        indigenous languages. </p>
```

Comment:
Always capitalize the "I":

```suggestion
        Indigenous languages. </p>
```

Reply:
Stole the text directly from the ALT Lab site - we should update it there too!